proposals/csharp-9.0/records.md

```diff
@@ -209,7 +209,9 @@ The first thing the constructor must do, is to call a copy constructor of the ba
 object constructor if the record inherits from object. An error is reported if a user-defined copy
 constructor uses an implicit or explicit constructor initializer that doesn't fulfill this requirement.
 After a base copy constructor is invoked, a synthesized copy constructor copies values for all instance
-fields implicitly or explicitly declared within the record type.
+fields implicitly or explicitly declared within the record type. 
+The sole presence of a copy constructor, whether explicit or implicit, doesn't prevent an automatic
+addition of a default instance constructor.
```

Comment:
Is a "default instance constructor" similar to what is synthesized for an empty class declaration `class C { }`?

Reply:
> Is a "default instance constructor" similar to what is synthesized for an empty class declaration class C { }?

It isn't similar, it is what is synthesized for an empty class declaration.